### PR TITLE
chore(MessageBox): unused keyframe

### PR DESCRIPTION
**Pull request by zhousg**

### CODE REVIEW

File: packages/theme-chalk/src/message-box.scss [hunk 1]
```diff
@@ -244,15 +244,3 @@
...
-  0% {
-    transform: translate3d(0, 0, 0);
-    opacity: 1;
-  }
-
-  100% {
-    transform: translate3d(0, -20px, 0);
-    opacity: 0;
-  }
-}
```
Comment (btea) [inline packages/theme-chalk/src/message-box.scss:258]: Perhaps it should not be removed, but applied in line 232.

Comment (zhousg) [reply packages/theme-chalk/src/message-box.scss:258]: `animation: msgbox-fade-in getCssVar('transition-duration') reverse;`  

But use reverse,  have less code

Comment (btea) [reply packages/theme-chalk/src/message-box.scss:258]: You are right!

**Review by btea — APPROVED** [review]

**Review by YunYouJun — APPROVED** [review]

**Comment by YunYouJun** [comment]
Yeah, it's unused.

Just for more information:

We use `msgbox-fade-in reverse` instead of `msgbox-fade-out`.

https://github.com/element-plus/element-plus/blob/e2167dcd55edccc9eec792c40772a040d3f42592/packages/theme-chalk/src/message-box.scss#L232

**Pull request merged**